frontend/leads-list/table/desktop-leads-table.component.tsx

```diff
@@ -37,7 +37,79 @@ export default function DesktopLeadsTable(props: LeadsTableProps) {
             </tr>
           ) : (
             props.leads.map(lead => {
-              console.log(lead);
+              function determineLeadStatus(
```

Comment:
small thing, but i think the component would be more readable if this function were pulled out of the jsx entirely -- maybe moved even outside of the component's function entirely. That way the JSX would still be pretty easily readable. Another smaller and less important reason is that when you put a function inside of another function (the arrow function for `props.lead.map(() => {})`), javascript actually has to recreate the function for each item, instead of being able to reuse the function. This is a negligible and not-actually-relevant performance hit. The bigger reason to change it is just for readability of the JSX.